config/locales/en/loan_questions.en.yml

```diff
@@ -37,19 +37,5 @@ en:
         edit_item: "Edit Item"
         delete_item: "Delete Item"
       form:
-        edit_item: "Edit Question or Group"
-        new_item: "New Question or Group"
-
-  simple_form:
```

Comment:
It does. We had simple-form in multiple spots referring to different items. So I'll go with that approach to keep similar items, like loan questions, together.